src/test/java/io/rcktapp/api/handler/sql/SqlGetHandlerTest.java

```diff
@@ -75,14 +75,20 @@ public class SqlGetHandlerTest {
     @Captor
     ArgumentCaptor<List> queryParametersCaptor;
 
+    @Captor
+    ArgumentCaptor<String> sqlCaptor;
+    
+    @Captor
+    ArgumentCaptor<Map<String, String>> rqlParamsCaptor;
+
```

Comment:
Looks like you added these variables, but they're not used? Or am I missing something?

Reply:
or, if they are, perhaps you've forgotten to delete in-line assignments to those references?